static/src/javascripts/projects/common/modules/commercial/creatives/template.js

```diff
@@ -56,6 +56,7 @@ define([
         this.params.logoguardian = svgs('logoguardian');
         this.params.marque36iconCreativeMarque = svgs('marque36icon', ['creative__marque']);
         this.params.logoFeatureLabel = config.switches.newCommercialContent ? 'Paid for by' : 'Brought to you by:';
+        this.params.isNewCommercialContent = !config.switches.newCommercialContent;
```

Comment:
it seems counter-intuitive to me, that two vars with the same name have opposite values